### PR TITLE
rename quo2

### PR DESCRIPTION
**Pull request by flexsurfer**
image

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 ### Redesign
 
-* Quo2, Switcher Navigation by @Parveshdhull in https://github.com/status-im/status-mobile/pull/13167
+* quo, Switcher Navigation by @Parveshdhull in https://github.com/status-im/status-mobile/pull/13167
```
Comment (flexsurfer) [inline RELEASES.md:28]: idk it doesn't actually matter

**Comment by J-Son89** [comment]
btw, have we updated the linting that checks whether we have used status-im/status-im2 code in quo2 directory? maybe we need to update that now?

**Comment by flexsurfer** [comment]
> btw, have we updated the linting that checks whether we have used status-im/status-im2 code in quo2 directory? maybe we need to update that now?

good point

File: src/quo/__visual_tests__/button.e2e.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ const loginToHomePage = async () => {
   await waitToNavigate(200);
   await element(by.id('browser-stack')).tap();
   await waitToNavigate(400);
-  await element(by.text('Quo2.0 Preview')).tap();
+  await element(by.text('Quo Preview')).tap();
```
Comment (ilmotta) [inline src/quo/__visual_tests__/button.e2e.js]: Does anybody use these *visual tests*? Shouldn't we remove `__visual_tests__` entirely?

**Review by ilmotta — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

File: src/quo/__visual_tests__/button.e2e.js (hunk 2 above)
Comment (J-Son89) [reply src/quo/__visual_tests__/button.e2e.js]: this should be removed for now yes, it was missed on the previous clean up

**Comment by flexsurfer** [comment]
> btw, have we updated the linting that checks whether we have used status-im/status-im2 code in quo2 directory? maybe we need to update that now?

done

File: src/quo/__visual_tests__/button.e2e.js (hunk 2 above)
Comment (flexsurfer) [reply src/quo/__visual_tests__/button.e2e.js]: oh cool , removing __visual_tests__ entirely

Comment (flexsurfer) [reply src/quo/__visual_tests__/button.e2e.js]: done

**Review by Parveshdhull — APPROVED** [review]

**Review by OmarBasem — APPROVED** [review]

**Review by siddarthkay — APPROVED** [review]

**Review by clauxx — APPROVED** [review]

**Pull request merged**